docs/quick-start.md

```diff
@@ -74,7 +76,12 @@ Follow the steps below to get up and running for the intended configuration elem
 5. Determine what data(variables) the Jinja2 templates need from Nautobot.
 
    1. See [Source of Truth Agg Details](./navigating-sot-agg.md)
-   2. Populate the SoTAgg field in the `Plugin -> Settings`.
+   2. In the UI `Extensibility -> GraphQL Queries`. Click Add.
+   3. Populate the GraphQL data.
+   4. Make sure to follow the format specified in
```

Comment:
specified in....
Is this missing the rest of the sentence?

Reply:
Looks like it. I'll get it fixed.